proposals/nnnn-actors.md

```diff
@@ -105,9 +117,9 @@ Semantically, the call to `accumulateInterest` is placed on the queue for the ac
 
 ### Global actors
 
-Actor classes provide a way to encapsulate state completely, ensuring that code outside the class (including other instances of the same actor class!) cannot access its mutable state. However, sometimes the code and mutable state isn't limited to a single class, for example, because it can only be accessed from the main thread or a UI thread.
+Actor classes provide a way to encapsulate state completely, ensuring that code outside the class cannot access its mutable state. However, sometimes the code and mutable state isn't limited to a single class. For example, in order to express the important concepts of "Main Thread" or "UI Thread" in this new Actor focused world we must be able to express and extend state and functions able to run on these specific actors even though they are not really all located in the same class. 
```

Comment:
We stressed the points about "including other instances of the same class" already in the previous section (I added a bit more focus on it) so I think we can skip the () here, wdyt (notice the additions above)